mico-core/src/main/java/io/github/ust/mico/core/model/MicoServiceDependency.java

```diff
@@ -60,6 +63,7 @@
     @JsonBackReference
```

Comment:
No, because the `service` is set by the backend code after it is retrieved from the database. Therefore the `service` property should not be part of the DTO that will be created in #512